src/test/javascript/spec/integration/code-editor/code-editor-container.integration.spec.ts

```diff
@@ -490,16 +478,17 @@ describe('CodeEditorContainerIntegration', () => {
         containerFixture.detectChanges();
 
         // saving before commit
-        expect(saveFilesStub).to.have.been.calledOnceWithExactly([{ fileName: unsavedFile, fileContent: 'lorem ipsum' }], true);
-        expect(container.editorState).to.equal(EditorState.SAVING);
-        expect(container.fileBrowser.status.editorState).to.equal(EditorState.SAVING);
+        expect(saveFilesStub).toHaveBeenCalledTimes(1);
+        expect(saveFilesStub).toHaveBeenCalledWith([{ fileName: unsavedFile, fileContent: 'lorem ipsum' }], true);
+        expect(container.editorState).toEqual(EditorState.SAVING);
+        expect(container.fileBrowser.status.editorState).toEqual(EditorState.SAVING);
         // committing
-        expect(commitStub).to.not.have.been.called;
-        expect(container.commitState).to.equal(CommitState.COMMITTING);
-        expect(container.fileBrowser.status.commitState).to.equal(CommitState.COMMITTING);
+        expect(commitStub).not.toHaveBeenCalled;
```

Comment:
```suggestion
        expect(commitStub).not.toHaveBeenCalled();
```
Same as above (The function call is omitted).

Reply:
Same.